src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const mainImage = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+thumbs.onclick = function (e) {
+  const thumbnail = e.target.closest('a');
```

Comment:
The use of 'closest' is appropriate here to ensure that the click event is captured even if the click is on a child element of the 'a' tag. However, ensure that the HTML structure supports this logic, meaning that the 'a' elements are indeed the closest ancestors of the clicked elements.